vscode/src/hover.ts

```diff
@@ -11,23 +11,21 @@ export function createHoverProvider(languageService: ILanguageService) {
 class QSharpHoverProvider implements vscode.HoverProvider {
   constructor(public languageService: ILanguageService) {}
 
-  async provideHover(
-    document: vscode.TextDocument,
-    position: vscode.Position,
-    // eslint-disable-next-line @typescript-eslint/no-unused-vars
-    token: vscode.CancellationToken,
-  ) {
+  async provideHover(document: vscode.TextDocument, position: vscode.Position) {
     const hover = await this.languageService.getHover(
       document.uri.toString(),
-      document.offsetAt(position),
+      position,
     );
     return (
       hover &&
       new vscode.Hover(
         new vscode.MarkdownString(hover.contents),
         new vscode.Range(
```

Comment:
Same here, can this be the toVscodeRange function?

Reply:
Yep, thanks for catching.